scripts/02-create-compose-env.sh

```diff
@@ -48,11 +48,6 @@ fi
 
 GPUS_LIST="$(join-list-contents ', ' `seq 0 $((NUM_GPUS-1))`)"
 
-CURRENT_CUDA_VERSION="11.5.0"
-if [[ "$(which nvcc)" != "" ]]; then
-    CURRENT_CUDA_VERSION="$(nvcc --version | head -n4 | tail -n1 | cut -d' ' -f5 | cut -d',' -f1)"
```

Comment:
Parsing is still good idea. May be add '.0' at the end to match with docker images.

Reply:
I remade this PR to append `.0`. It's probably good enough as a fix for now.